pom.xml

```diff
@@ -377,6 +426,14 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>findbugs-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
```

Comment:
@chrishantha Is it necessary to generate javadocs and sources everytime? This adds to the build time of our repos. Please note that maven-deploy-plugin always generate javadocs and sources during releases. That's where these plugins are most important. @jsdjayanga, please review.